signing_service/signing_service/tests/tests.py

```diff
@@ -4,11 +4,12 @@
 
 import mock
 
-from ..constants import SIGNING_SERVICE_DEFAULT_PORT
-from ..constants import SIGNING_SERVICE_RECOVERABLE_ERRORS
-from ..constants import SIGNING_SERVICE_MAXIMUM_RECONNECT_TIME
-from ..signing_service import _parse_arguments
-from ..signing_service import SigningService
+from signing_service.constants import SIGNING_SERVICE_DEFAULT_PORT
+from signing_service.constants import SIGNING_SERVICE_RECOVERABLE_ERRORS
+from signing_service.constants import SIGNING_SERVICE_MAXIMUM_RECONNECT_TIME
+from signing_service.exceptions import SigningServiceValidationError
```

Comment:
This import also should not be in this commit. You are adding validation to tests.

Reply:
It was previously on line 9.

`from exceptions import SigningServiceValidationError`